src/GoodsList.tsx

```diff
@@ -8,7 +8,11 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
```

Comment:
Make sure that you add some rendering optimization techniques (React.memo and so on.)